app/controllers/manager/students/profiles_controller.rb

```diff
@@ -15,8 +15,9 @@ def show
         @exposure = metrics_service.exposure
         @survey_level = metrics_service.calculate_survey_level
 
-        profile_metrics_service = ::Students::ProfileMetricsService.new(@student_profile)
-        @profile_level = profile_metrics_service.calculate_profile_level
+        #profile_metrics_service = ::Students::ProfileMetricsService.new(@student_profile)
+        #@profile_level = profile_metrics_service.calculate_profile_level
```

Comment:
Porque isso foi comentado?

Reply:
É uma mudança provisória, foi discutido em uma daily no inicio assim que iniciei a tarefa, que o calculo do nivel do aluno ainda não estava 100% pronto e que eu poderia inserir o nível manualmente, por isso foi comentado essa linha e inserido essa  @profile_level = @student_profile.level para que o sistema entendesse que o nível do aluno seria definido manualmente.

Reply:
Não se comenta código. Se realmente algo no futuro for acontecer ele será criado com comentário ou sem comentário, lembre-se Yagni